lxqt-base/lxqt-policykit/lxqt-policykit-0.17.0.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="LXQt PolKit authentication agent"
+HOMEPAGE="https://lxqt.github.io/"
+
+if [[ ${PV} = *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/lxqt/${PN}.git"
+else
+	SRC_URI="https://github.com/lxqt/${PN}/releases/download/${PV}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
+fi
+
+LICENSE="LGPL-2.1 LGPL-2.1+"
+SLOT="0"
+
+BDEPEND="
+	dev-qt/linguist-tools:5
+	>=dev-util/lxqt-build-tools-0.9.0
+	virtual/pkgconfig
+"
+DEPEND="
+	dev-libs/glib:2
+	>=dev-libs/libqtxdg-3.3.1
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtwidgets:5
+	=lxqt-base/liblxqt-$(ver_cut 1-2)*
+	sys-auth/polkit-qt[qt5(+)]
```

Comment:
Optional: Drop `[qt5(+)]` by depending on `>=sys-auth/polkit-qt-0.113.0`, our first version to have dropped Qt4.